doc/classes/Shape2D.xml

```diff
@@ -50,7 +52,9 @@
 			<argument index="3" name="shape_xform" type="Transform2D" />
 			<argument index="4" name="shape_motion" type="Vector2" />
 			<description>
-				Returns a list of the points where this shape would touch another, if a given movement was applied. If there are no collisions the list is empty.
+				Returns a list of contact point pairs where this shape would touch another, if a given movement was applied.
+				If there would be no collisions the list is empty. Otherwise, the list contains contact points arranged in pairs, with entries alternating between points on the boundary of this shape and points on the boundary of [code]with_shape[/code].
+				A collision pair A, B can be used to calculate the collision normal with [code](B-A).normalized()[/code], and the collision depth with [code](B-A).length()[/code]. This information is typically used to separate shapes, particularly in collision solvers.
```

Comment:
```suggestion
				Returns a list of contact point pairs where this shape would touch another, if a given movement was applied.
				If there would be no collisions, the returned list is empty. Otherwise, the returned list contains contact points arranged in pairs, with entries alternating between points on the boundary of this shape and points on the boundary of [code]with_shape[/code].
				A collision pair A, B can be used to calculate the collision normal with [code](B - A).normalized()[/code], and the collision depth with [code](B - A).length()[/code]. This information is typically used to separate shapes, particularly in collision solvers.
```